CHANGELOG.md

```diff
@@ -6,6 +6,32 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased 1.1.z]
 
+## [1.1.10] - 2023-10-30
+
+> Śruba, przykręcona we śnie, nie zmieni sytuacji, jaka panuje na jawie.
+
+### Security
+
+* Fix a semi-arbitrary cgroup write bug when given a malicious hugetlb
+  configuration. As discussed in the [security advisory][cve-2023-45804], our
```

Comment:
the CVE has been revoked, so instead we can link to the advisory for details (which can be viewed after we publish it)

Reply:
In fact, the text does not mention CVE, it is just a label.

Reply:
I opened a dummy issue (#4103) for the advisory and have added a reference to it.